src/style.css

```diff
@@ -1,3 +1,108 @@
+/* Reset de margens padrão do navegador */
+body, h1, h2, h3, h4, h5, h6, p, ul, ol, li, figure, figcaption, blockquote, dl, dd {
+  margin: 0;
+}
+
+/* Importação da fonte Roboto */
 body {
+  background-color: rgb(230, 227, 227);
+  font-family: 'Roboto', sans-serif;
+  font-weight: bold;
+  display: flex;
+  justify-content: center; /* Centraliza horizontalmente */
+  align-items: center; /* Centraliza verticalmente */
+}
+
+/* Estilo do header */
+header {
+  width: 1200px;
```

Comment:
Header should stretch to page's full width regardless of the page size [Reference](https://github.com/mate-academy/layout_moyo-header#:~:text=the%20header%20should%20stretch%20the%20full%20page%20width)